src/wxchartaxisoptions.cpp

```diff
@@ -22,30 +22,48 @@
 
 #include "wxchartaxisoptions.h"
 
-wxChartAxisOptions::wxChartAxisOptions(wxChartAxisPosition position)
-    : m_position(position), m_labelType(wxCHARTAXISLABELTYPE_POINT),
-    m_startMarginType(wxCHARTAXISMARGINTYPE_NONE), m_endMarginType(wxCHARTAXISMARGINTYPE_NONE),
-    m_startValueMode(wxCHARTAXISVALUEMODE_AUTO), m_startValue(0),
-    m_endValueMode(wxCHARTAXISVALUEMODE_AUTO), m_endValue(0),
-    m_lineWidth(1), m_lineColor(0, 0, 0, 0x19), m_overhang(3),
-    m_fontOptions(wxFONTFAMILY_SWISS, 12, wxFONTSTYLE_NORMAL, wxFONTWEIGHT_NORMAL, 0x666666)
-{
-}
-
-wxChartAxisOptions::wxChartAxisOptions(wxChartAxisPosition position,
+wxChartAxisOptions::wxChartAxisOptions(wxChartAxisLocation location)
```

Comment:
Changed "position" to "location", as i kept confusing the "GetPosition()" Axis / AxisOptions functions.